net-libs/libsearpc/libsearpc-3.3.0.ebuild

```diff
@@ -29,6 +29,10 @@ RDEPEND="${DEPEND}
 
 S="${WORKDIR}/${PN}-${RELEASE_COMMIT}"
```

Comment:
git mv needed for implicit func. decl fixes, as it can affect runtime (see https://wiki.gentoo.org/wiki/Project:Toolchain/Compiler_warnings)

Reply:
Fixed.